### PR TITLE
Remove masslimits qcd unc

### PR DESCRIPTION
**Pull request by valeriadamante**
This PR has many purposes, which I will summarise here:
- remove higher limits for H(bb) and H(ττ) masses as agreeded
- add QCD uncertainty (both scale and norm, but currently implemented only scale)
- define correctly the selection for all the channels which are not tauTau in order to produce anaTuples containing them

### CODE REVIEW

File: AnaProd/tasks.py [hunk 1]
```diff
@@ -264,7 +264,10 @@ def create_branch_map(self):
         branches = {}
         anaProd_branch_map = AnaTupleTask.req(self, branch=-1, branches=()).branch_map
         for br_idx, (sample_id, sample_name, sample_type, input_file) in anaProd_branch_map.items():
+            #if sample_type == 'QCD':continue
+            #if br_idx == 600: continue
             branches[br_idx] = (sample_name, sample_type)
+            #print(branches)
```
Comment (kandrosov) [inline AnaProd/tasks.py:270]: undo changes

File: Analysis/MassCut.py
Comment (kandrosov) [inline Analysis/MassCut.py:1]: move to Studies

File: AnaProd/HH_bbtautau/baseline.py [hunk 2]
```diff
@@ -75,16 +76,16 @@ def RecoHttCandidateSelection(df, config):
...
-        Muon_B0 && ( (Muon_tightId && Muon_pfRelIso04_all < 0.15) || (Muon_highPtId && Muon_tkRelIso < 0.15) )
+        Muon_B0 && ( (Muon_tightId && Muon_pfRelIso04_all < 0.15) )
     """)
     df = df.Define("Muon_B2_muMu_2", f"""
-        Muon_B0 && ( (Muon_tightId && Muon_pfRelIso04_all < 0.3) || (Muon_highPtId && Muon_tkRelIso < 0.3) )
+        Muon_B0 && ( (Muon_tightId && Muon_pfRelIso04_all < 0.3) )
     """)
 
     df = df.Define("Electron_B2_eMu_1",f"Electron_B0 && Electron_mvaIso_WP80 ")
     #  Electron_mvaNoIso_WP80 && Electron_pfRelIso03_all < 0.3
```
Comment (kandrosov) [inline AnaProd/HH_bbtautau/baseline.py:86]: - do we store `mvaNoIso_WP80` and `mvaIso_WP80` for electron in anaTuple? If not, we should.
- for eMu I would keep OR for electron selection (and apply the final choice at the anaTuple level)

**Review by kandrosov — CHANGES_REQUESTED** [review]

File: AnaProd/HH_bbtautau/baseline.py (hunk 2 above)
Comment (valeriadamante) [reply AnaProd/HH_bbtautau/baseline.py]: we store them
ok I make this change

File: AnaProd/HH_bbtautau/baseline.py [hunk 3]
```diff
@@ -75,16 +76,16 @@ def RecoHttCandidateSelection(df, config):
...
-        Muon_B0 && ( (Muon_tightId && Muon_pfRelIso04_all < 0.15) || (Muon_highPtId && Muon_tkRelIso < 0.15) )
+        Muon_B0 && ( (Muon_tightId && Muon_pfRelIso04_all < 0.15) )
     """)
     df = df.Define("Muon_B2_muMu_2", f"""
-        Muon_B0 && ( (Muon_tightId && Muon_pfRelIso04_all < 0.3) || (Muon_highPtId && Muon_tkRelIso < 0.3) )
+        Muon_B0 && ( (Muon_tightId && Muon_pfRelIso04_all < 0.3) )
     """)
 
-    df = df.Define("Electron_B2_eMu_1",f"Electron_B0 && Electron_mvaIso_WP80 ")
+    df = df.Define("Electron_B2_eMu_1",f"Electron_B0 && (Electron_mvaIso_WP80 ||  Electron_mvaNoIso_WP80 && Electron_pfRelIso03_all < 0.3) ")
```
Comment (kandrosov) [inline AnaProd/HH_bbtautau/baseline.py:85]:
```suggestion
    df = df.Define("Electron_B2_eMu_1",f"Electron_B0 && (Electron_mvaIso_WP80 || (Electron_mvaNoIso_WP80 && Electron_pfRelIso03_all < 0.3)) ")
```

**Review by kandrosov — CHANGES_REQUESTED** [review]

File: inference
Comment (kandrosov) [inline inference:1]: don't update inference within this PR

**Review by kandrosov — CHANGES_REQUESTED** [review]

**Review by kandrosov — APPROVED** [review]

**Pull request merged**